src/compile/layer.ts

```diff
@@ -233,6 +233,7 @@ export class LayerModel extends Model {
   }
 
   protected mapping() {
+    // TODO: what is this?
```

Comment:
This is null for layer, concat, repeat.   It should only exists for `facet` and `unit`.